app/src/main/java/com/kickstarter/services/KSApolloClientV2.kt

```diff
@@ -380,42 +388,38 @@ class KSApolloClientV2(val service: ApolloClient, val gson: Gson) : ApolloClient
         }
     }
 
-    override fun getStoredCards(): Observable<List<StoredCard>> {
-        return Observable.defer {
-            val ps = PublishSubject.create<List<StoredCard>>()
+    override suspend fun _getStoredCards(): List<StoredCard> {
+        val query = UserPaymentsQuery()
+        var ret = listOf<StoredCard>()
+        try {
+            // `single()` vs `singleOrNull()`
+            val response = this.service.query(query).toFlow().single()
```

Comment:
could be changed over `val response = this.service.query(query).execute()` it internally uses `toFlow().single()`